atom/browser/native_window_views_win.cc

```diff
@@ -90,6 +90,15 @@ bool NativeWindowViews::PreHandleMSG(
     // mode if it isn't already, always say we didn't handle the message
     // because we still want Chromium to handle returning the actual
     // accessibility object.
+    case WM_DWMCOLORIZATIONCOLORCHANGED: {
```

Comment:
@zcbenz That works, thanks.

However, after some more exploration it looks like a message only window won't do the job as 

> does not receive broadcast messages

https://msdn.microsoft.com/en-us/library/windows/desktop/ms632599.aspx#message_only